pages/home.js

```diff
@@ -32,7 +32,14 @@ export default function Home(props) {
       id="homeContent"
       className="container grid grid-cols-1 gap-y-5 mx-auto sm:flex sm:justify-center sm:gap-x-5"
     >
-      <HomeCardContainer title={t.createRoomTitle} desc={t.createRoomDesc}>
+      <HomeCardContainer>
+        <h1 className="text-opacity-75 text-black font-bold text-2xl">
```

Comment:
Since the `Scrum Poker` banner is an H1, I think these should be `H2`s and the descriptions should be `H3`s so there is a logical page order